sigs.yaml

```diff
@@ -3467,6 +3467,47 @@ workinggroups:
     liaison:
       github: pohly
       name: Patrick Ohly
+- dir: wg-serving
+  name: Serving
+  mission_statement: >
+    Discuss and enhance the support of inference serving for accelerated workloads.
```

Comment:
Do we need to limit the focus to accelerated workloads only?

Reply:
second sentence now covers other scenarios too. I think concentrating on accelerators would make sense. Are you thinking of CPU serving as a fallback option as a scenario? Or pure CPU serving that needs to be improved?